installer/data/template/.origintrail_noderc_one_click_testnet.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
do we need 

"auth": {
    "ipWhitelist": [
      "::1",
      "127.0.0.1",
    ]
  } ?

Reply:
it's removed